code/controllers/subsystem/dynamic/dynamic.dm

```diff
@@ -201,6 +201,55 @@ SUBSYSTEM_DEF(dynamic)
+
+			if (show_core_territory)
+				advisory_string += "Advisory Level: <b>Blue Star</b></center><BR>"
+				advisory_string += "Your sector's advisory level is Blue Star. At this threat advisory, the risk of attacks on Nanotrasen assets within the sector is minor but cannot be ruled out entirely. Remain vigilant."
+			else
+				advisory_string += "Advisory Level: <b>Green Star</b></center><BR>"
+				advisory_string += "Your sector's advisory level is Green Star. Surveillance information shows no credible threats to Nanotrasen assets within the Spinward Sector at this time. As always, the Department of Intelligence advises maintaining vigilance against potential threats, regardless of a lack of known threats."
+		if(20 to 39)
+			advisory_string += "Advisory Level: <b>Yellow Star</b></center><BR>"
+			advisory_string += "Your sector's advisory level is Yellow Star. Surveillance shows a credible risk of enemy attack against our assets in the Spinward Sector. We advise a heightened level of security alongside maintaining vigilance against potential threats."
+		if(40 to 65)
+			advisory_string += "Advisory Level: <b>Orange Star</b></center><BR>"
+			advisory_string += "Your sector's advisory level is Orange Star. Upon reviewing your sector's intelligence, the Department has determined that the risk of enemy activity is moderate to severe. At this advisory, we recommend maintaining a higher degree of security and reviewing red alert protocols with command and the crew."
+		if(66 to 79)
+			advisory_string += "Advisory Level: <b>Red Star</b></center><BR>"
+			advisory_string += "Your sector's advisory level is Red Star. The Department of Intelligence has decrypted Cybersun communications suggesting a high likelihood of attacks on Nanotrasen assets within the Spinward Sector. Stations in the region are advised to remain highly vigilant for signs of enemy activity and to be on high alert."
+		if(80 to 99)
+			advisory_string += "Advisory Level: <b>Black Orbit</b></center><BR>"
+			advisory_string += "Your sector's advisory level is Black Orbit. Your sector's local communications network is currently undergoing a blackout, and we are therefore unable to accurately judge enemy movements within the region. However, information passed to us by GDI suggests a high amount of enemy activity in the sector, indicative of an impending attack. Remain on high alert and vigilant against any other potential threats."
+		if(100)
+			advisory_string += "Advisory Level: <b>Midnight Sun</b></center><BR>"
+			advisory_string += "Your sector's advisory level is Midnight Sun. Credible information passed to us by GDI suggests that the Syndicate is preparing to mount a major concerted offensive on Nanotrasen assets in the Spinward Sector to cripple our foothold there. All stations should remain on high alert and prepared to defend themselves."
+	return advisory_string
+
+/proc/the_real_threat_level(threat)
+	var/admin_threat_string = ""
+	switch(round(threat))
```

Comment:
```suggestion

/datum/controller/subsystem/dynamic/proc/get_real_threat_level()
	switch(round(threat_level))
```

Also now that I think about it, `shown_threat` that gets passed into this proc as an argument is a property of this subsystem already.
...but don't you actually want `threat_level` and not `shown_threat` anyway?

Reply:
I think you're right, and I think I've done all youre suggested changed but let me test it on local before merging; since you've shook my coding confidence to the ground :P